src/rules/stylelint-disable-reason/README.md

```diff
@@ -0,0 +1,127 @@
+# stylelint-disable-reason
+
+Control comments should be preceded by a comment explaining why stylelint are being disabled.
+
+```css
+a {
+  /* Need for IE 6 */                   /*←*/
+  /* eslint-disable no-browser-hacks */ /*↑*/
+  /* Need for IE 6 */                   /*←*/
+  _display: block;                      /*↑*/
+  /* eslint-enable no-browser-hacks */  /*↑*/
+}                                       /*↑*/
+/**                                       ↑
+*                           The reason of disable stylelint */
+```
```

Comment:
Can we extend this rule to also deal with `stylelint-disable-line` commands?

With `"preceding"`, the following would trigger a warning:

```js
a {
  color: pink; /* stylelint-disable-line color-no-named */
}
```

The following would not be a warning:

```js
a {
  /* Special case for pink here */
  color: pink; /* stylelint-disable-line color-no-named */
}
```